src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,250 @@
+package duke;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.ToDo;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.lang.String;
+import java.util.Scanner;
+import java.util.ArrayList;
+
+import static java.util.stream.Collectors.toList;
+
+public class Duke {
+
+    public static ArrayList<Task> tasks = new ArrayList<>();
+
+    public static void main(String[] args) {
+        displayDuke();
+        helloMessage();
+        commandsAvailable();
+
+        int listIndex = 0;
+
+        File file = new File("duke.txt");
+        listIndex = readFromFile(file, listIndex);
+
+        Scanner input = new Scanner(System.in);
+
+        while (true) {
+            String userInput = input.nextLine();
+            String[] commandGiven = userInput.split(" ");
+
+            try {
+                if (userInput.toLowerCase().trim().equals("list")) {
+                    drawLines();
+                    System.out.print("Here" + ((listIndex > 1) ? " are" : " is") + " the");
+                    System.out.println(((listIndex > 1) ? " tasks" : " task") + " in the list");
+                    for (int j = 0; j < listIndex; j++) {
+                        System.out.println((j + 1) + "." + tasks.get(j).toString());
+                    }
+                    drawLines();
+                } else if (userInput.toLowerCase().trim().equals("todo")) {
+                    drawLines();
```

Comment:
Is there any way to avoid the duplicated code of wrapping your Strings between lines?